python/tvm/micro/contrib/zephyr.py

```diff
@@ -204,6 +204,22 @@ def library(self, output, sources, options=None):
         )
         return tvm.micro.MicroLibrary(build_dir, [f"lib{project_name}.a"])
 
+    def _print_make_statistics(self, output):
+        print("==================")
+        output = output.splitlines()
+        lines = iter(output)
+        for line in lines:
+            if line.startswith("Memory region"):
+                # print statistics header
+                print(line)
```

Comment:
it might be better to use _LOG.info() rather than print, so it could be suppressed if needed

Reply:
done. @areusch Thanks a lot for the review.[